x-pack/plugins/lens/public/xy_visualization/xy_config_panel.tsx

```diff
@@ -4,11 +4,11 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import './xy_config_panel.scss';
```

Comment:
Can we document this pattern of putting the style import at the very top of the component file somewhere and make it an official convention?

Reply:
Added to .md file https://github.com/elastic/kibana/pull/71444/commits/1eff88890633b6e7a8f7d4b10bebdeb35d49e4ad

Reply:
Thanks a lot, it’s done differently in a bunch of places and it’s always good to have an official way of doing things 🙂